src/Adapter/MSTest.TestAdapter/Execution/TestClassInfo.cs

```diff
@@ -108,6 +111,24 @@ internal set
     /// </summary>
     internal Dictionary<MethodInfo, int> ClassInitializeMethodTimeoutMilliseconds { get; }
 
+    /// <summary>
+    /// Gets the timeout for the class cleanup methods.
+    /// We can use a dictionary because the MethodInfo is unique in an inheritance hierarchy.
+    /// </summary>
+    internal Dictionary<MethodInfo, int> ClassCleanupMethodTimeoutMilliseconds { get; }
+
+    /// <summary>
+    /// Gets the timeout for the test initialize methods.
+    /// We can use a dictionary because the MethodInfo is unique in an inheritance hierarchy.
+    /// </summary>
+    internal Dictionary<MethodInfo, int> TestInitializeMethodTimeoutMilliseconds { get; private set; }
+
+    /// <summary>
+    /// Gets the timeout for the test cleanup methods.
+    /// We can use a dictionary because the MethodInfo is unique in an inheritance hierarchy.
+    /// </summary>
+    internal Dictionary<MethodInfo, int> TestCleanupMethodTimeoutMilliseconds { get; private set; }
```

Comment:
These 2 dictionaries seem to be initialized only from ctor so the properties should be readonly instead.